ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -65,20 +84,71 @@ open class CommandBar: UIView {
     public override func layoutSubviews() {
         super.layoutSubviews()
 
+        commandBarContainerStackView.layoutIfNeeded()
+
         containerMaskLayer.frame = containerView.bounds
         updateShadow()
     }
 
+    /// Scrollable items shown in the center of the CommandBar
+    public var itemGroups: [CommandBarItemGroup] {
+        get {
+            mainCommandGroupsView.itemGroups
+        }
+        set {
+            mainCommandGroupsView.itemGroups = newValue
+        }
+    }
+
+    /// Items pinned to the leading end of the CommandBar
+    public var leadingItemGroups: [CommandBarItemGroup]? {
```

Comment:
Not necessarily. However, I feel like having a tiny bit of additional handling logic here and being able to do `commandBar. leadingItemGroups = nil` to clear the items is better than the alternative of making consumers of the `CommandBar` call `commandBar.leadingItemGroups.removeAll()`, `commandBar.leadingItemGroups = []` or exposing a `CommandBar.removeLeadingItemGroups()` instance method. I guess having to set an array to an empty array for a public API seems clunky to me. I'm open to a further discussion on this. I may be missing something.

Reply:
Actually, after playing around with this a bit more it's not so bad. I will update these to be non-optional.

Reply:
Fwiw I am for using nil instead of an empty object wherever possible. There might not be a distinction now, but in these cases there often arises a distinction where empty means something different than nil from the consumer's perspective. And I think storing no object technically should use less memory than storing an empty object.

If the if let is causing code to seem overly nested, I would rather focus on other factoring we can do to reduce some of the other layers of nesting. If there is other rationale I'm not thinking of, would love to hear that (for my education).

Reply:
I guess my main point was to keep the 2 APIs consistent (either both are optionals or both are non-optionals).

It seems like the **CommandBarCommandGroupsView** class could have a nullable **itemGroups** property.